lib/data_loadable.rb

```diff
@@ -0,0 +1,12 @@
+require 'csv'
```

Comment:
This looks great. Good job spotting something to be abstracted out & good use of module/class var's.